catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/commands/SearchFormsLoaderCommandTest.java

```diff
@@ -0,0 +1,132 @@
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import com.google.common.collect.ImmutableList;
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Metacard;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.net.URL;
+import java.util.Collections;
+import org.codice.ddf.catalog.ui.forms.SearchFormsLoader;
+import org.codice.ddf.catalog.ui.util.EndpointUtil;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+
+@RunWith(JUnit4.class)
+public class SearchFormsLoaderCommandTest {
+  private @Mock SearchFormsLoader mockLoader;
+
+  private @Mock Metacard mockMetacard1;
+
+  private @Mock Metacard mockMetacard2;
+
+  private static final URL LOADER_RESOURCES_DIR =
+      SearchFormsLoaderCommandTest.class.getResource("/forms/loader");
+
+  private static final String ROOT = LOADER_RESOURCES_DIR.getPath();
+
+  private static ByteArrayOutputStream buffer;
+
+  private static PrintStream realSystemOut;
+
+  private SearchFormsLoaderCommand cmd =
+      new SearchFormsLoaderCommand() {
+        @Override
+        public SearchFormsLoader generateLoader(
+            CatalogFramework cf, EndpointUtil util, String dir, String forms, String results) {
+          return mockLoader;
+        }
+      };
+
+  @BeforeClass
+  public static void setupOutput() {
+    interceptSystemOut();
+  }
+
+  @Before
+  public void setup() {
+    initMocks(this);
+    when(mockMetacard1.getId()).thenReturn("1");
+    when(mockMetacard2.getId()).thenReturn("2");
+    when(mockMetacard1.getTitle()).thenReturn("title1");
+    when(mockMetacard2.getTitle()).thenReturn("title2");
+
+    when(mockLoader.retrieveSystemTemplateMetacards())
+        .thenReturn(ImmutableList.of(mockMetacard1, mockMetacard2));
+  }
+
+  @After
+  public void resetBuffer() {
+    buffer.reset();
+  }
+
+  @AfterClass
+  public static void closeOutput() throws IOException {
+    System.setOut(realSystemOut);
+    buffer.close();
+  }
+
+  @Test
+  public void testLoadMetacardsFromDirectory() {
+    cmd.formsDir = ROOT + "/valid";
+    cmd.formsFile = "forms.json";
+    cmd.resultsFile = "results.json";
+
+    cmd.executeWithSubject();
+    assertThat(this.getOutput(), containsString("Initializing Search Form Template Loader"));
+    assertThat(
+        this.getOutput(),
+        containsString("Loader initialized, beginning ingestion of system templates."));
+    assertThat(this.getOutput(), containsString("System templates successfully ingested."));
+    verify(mockLoader, times(1)).retrieveSystemTemplateMetacards();
+    verify(mockLoader, times(1)).bootstrap(refEq(ImmutableList.of(mockMetacard1, mockMetacard2)));
```

Comment:
I don't think you need `refEq` here. Without an argument matcher, Mockito will simply check that the two lists are equal, which they are.